### ISSUE TITLE
Add status message for clause addition/removal in Profiler

### ISSUE THREAD

**Issue by smartguest** [microsoft/azuredatastudio#18578]
Accessibility testing has shown that there is no status message that appears when a clause has been added or removed, this makes it difficult for users without vision to know if something has been added or removed. A status message that appears when clauses actions have been started should be implemented.
